src/interfaces/uproxy_core_api.ts

```diff
@@ -215,14 +217,29 @@ export interface InvitationData {
 
 export enum PortControlSupport {PENDING, TRUE, FALSE};
 
-// Argument to #cloudInstall.
-export interface CloudInstallArgs {
-  // Use this cloud computing provider to create a server.
-  providerName: string;
-  // Provider-specific region in which to locate the new server.
-  region: string;
+export enum CloudOperationType {
+  CLOUD_INSTALL = 0,
+  CLOUD_DESTROY = 1
+}
+
+// Arguments to cloudUpdate
+export interface CloudOperationArgs {
+  operation: CloudOperationType;
+  // Use this cloud computing provider to access a server.
+  providerName :string;
+  // Provider-specific region in which to locate a new server.
+  region ?:string;
```

Comment:
OK, we can leave this as is for now.  Once we have a few more cloud operations added, we can revisit this if we find they all need very different params.